nix/ocaml.nix

```diff
@@ -170,6 +171,7 @@ let
           mv _build/default/src/app/cli/src/mina.exe $out/bin/mina
           mv _build/default/src/app/logproc/logproc.exe $out/bin/logproc
           mv _build/default/src/app/rosetta/rosetta.exe $out/bin/rosetta
+          mv _build/default/src/app/batch_txn_tool/batch_txn_tool.exe $out/bin/batch_txn_tool
```

Comment:
Is this a tool that's used often? If not, it should probably be moved to a different output

Reply:
I think this tool is only used locally, and for now on the CI we only check that it builds. So I moved it to a different output.